docs/ADDITIONAL_INFORMATION.md

```diff
@@ -0,0 +1,14 @@
+Additional Information
+## Release cadence
+<release cadence>
+
+## Versioing
```

Comment:
Please correct this misspelling:   Versioing  => Versioning

Reply:
Addressed as suggested